src/fides/api/ops/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -230,3 +245,97 @@ def save_privacy_preferences(
     consent_request.save(db=db)
 
     return upserted_current_preferences
+
+
+@router.get(
+    CURRENT_PRIVACY_PREFERENCES,
+    status_code=HTTP_200_OK,
+    dependencies=[
+        Security(verify_oauth_client, scopes=[CURRENT_PRIVACY_PREFERENCE_READ])
+    ],
+    response_model=Page[CurrentPrivacyPreferenceReportingSchema],
+)
+def get_current_privacy_preferences(
+    *,
+    params: Params = Depends(),
+    db: Session = Depends(get_db),
+    created_lt: Optional[datetime] = None,
+    created_gt: Optional[datetime] = None,
```

Comment:
I ran this format by Michael - he said largely folks will be accessing the historical report.  If we want to make it look like the "historical" report, we would join in the PrivacyPreferenceHistory table.  I'm leaving this for now because I don't think we're very opinionated about this yet but will revisit in the consent reporting wrap up ticket.

Reply:
@adamsachs the issue is that I don't think this endpoint should be returning the entirety of the nested PrivacyPreferenceHistory and the PrivacyNoticeHistory records.  I assume in the end I may be just pulling in a subset of attributes from these other records, and returning them in a flattened form, like I am in the Historical Report.   This may populate a table in the UI or a spreadsheet.

Michael says it's fine to leave is as-is, and finalize the specific fields here when we have more customer insight.

Reply:
ok! thanks for explaining that thoroughly, it makes sense.